### PR TITLE
align Pratt parser error messages

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: src/prattparser.js [hunk 1]
```diff
@@ -8,7 +8,10 @@ var assert = require('assert');
 var {isString} = require('./type-utils');
 var {SyntaxError, TemplateError} = require('./error');
 
-let syntaxRuleError = (token, expects) => new SyntaxError(`Found '${token.value}' expected '${expects}'`, token);
+let syntaxRuleError = (token, expects) => {
+  expects.sort();
```
Comment (alexandrasp) [inline src/prattparser.js]: Is it necessary to sort? Do values from tokens come sorted?

File: test/test_prattparser.py [hunk 2]
```diff
@@ -104,7 +104,7 @@ def fail(input, message):
             eq_(str(exc), "SyntaxError: " + message)
 
     yield fail, 'x', 'Unexpected input: \'x\''
-    yield fail, '11', 'Found number, expected *, +, -'
+    yield fail, '11', 'Found 1, expected *, +, -'
```
Comment (alexandrasp) [inline test/test_prattparser.py]: maybe 1 should be 11, or the opposite. :)

**Comment by djmitche** [comment]
I changed the "11" to "12".  Anything else to look at?

**Comment by djmitche** [comment]
Oops, I should have re-requested review.  I think this is ready to go?

File: src/prattparser.js (hunk 1 above)
Comment (alexandrasp) [reply src/prattparser.js]: Ok.

**Comment by alexandrasp** [comment]
Ready to go, @djmitche.

**Pull request merged**